src/WebJobs.Script/Description/Workers/WorkerFunctionInvoker.cs

```diff
@@ -140,7 +140,9 @@ private async Task<(string name, DataType type, object value)[]> BindInputsAsync
                     return (binding.Metadata.Name, bindingContext.DataType, bindingContext.Value);
                 });
 
-            return await Task.WhenAll(bindingTasks);
+            var result = await Task.WhenAll(bindingTasks);
+            _logger.LogInformation($"InputInformation: {binder.GetInformation()}");
+            return result;
```

Comment:
`_logger` is for system logs. Looking at the PR in webjobs sdk, this needs to be logged to user logger `context.logger` in ScriptInvocationContext.

Also, change to `LogTrace` as this is purely for debugging.

Reply:
Just to make sure, using `LogTrace` with the `context.Logger` will show up in telemetry on Kusto right?

Reply:
I am concerned about logging this info to system logs. cc @brettsam

Reply:
I changed in the next commit - not using `_logger` anymore. Using `context.Logger.LogTrace` (the expectation is to get these logs in Kusto.)

Reply:
When you refer to seeing logs in Kusto - are you referring to system logs? Logging using context.Logger.LogTrace will log to Application Insights. Only users will have access to these logs

Reply:
We want this in system logs too. We had a discussion with @paulbatum about this but didn't specifically talk about whether it should go to Kusto or App Insights+Kusto. Any thoughts?